tox.ini

```diff
@@ -15,3 +16,8 @@ commands = multilint
 
 [testenv:py35-codestyle]
 commands = multilint
+
+[testenv:py35-numpy]
+deps = -r{toxinidir}/requirements.txt
+    numpy
```

Comment:
don't do this, put it in `requirements.in` and then run `pip-compile -r -U` to update `requirements.txt` (`pip install pip-tools` for `pip-compile`)